packages/core/src/tools/LinkingProvider.ts

```diff
@@ -123,6 +128,17 @@ class LinkingProvider {
       message: `Error while opening ${address} in the map app`,
     });
   }
+
+  async openGoogleMapsDirections(
+    pois: Array<{address: string; latitude?: number; longitude?: number}>,
+  ) {
+    const url = createGoogleMapsDirectionsURL(pois);
+
+    await this.openURL(url, {
+      title: 'Could not open the map app',
```

Comment:
Maybe precise that you want to open google map. Also when you don't have google map app it opens default web browser, so maybe you need to change the wording